Geometry_Engine/Query/IsEqual.cs

```diff
@@ -35,6 +36,7 @@ public static partial class Query
         /**** public Methods - Vectors                 ****/
         /***************************************************/
 
+        [Description("Checks if the two Planes are geometrically equal, i.e. if all of their properties are within tolerance of each other.")]
```

Comment:
Not sure here. @FraserGreenroyd ?

Reply:
I might argue a `within tolerance with each other.` However, I would also suggest that maybe the sentence as a whole be reworked.

`Checks if the two Planes are geometrically equal, i.e. if all of their properties match within a given tolerance.`

I don't think the `each other` bit is necessarily necessary owing to this method only checking two planes (based on the description, I have not read the code in full), so that it is doing it against each other is already a given I would think?

Reply:
Taken! 👍